docs/static/generated/rest_v1_dispatcher.yml

```diff
@@ -2304,7 +2304,7 @@ components:
           type: integer
           format: int64
         plan:
-          type: string
```

Comment:
🤔 do we also want to have "any" here instead?

Reply:
`any` isn't valie in the openapi spec

Reply:
ok, what causes confusion to me is that we document it as any in the htlm docs; is there any way we can document is as 'object' there?

Reply:
> is there any way we can document is as 'object' there?

Yes; I've pushed a change for this.